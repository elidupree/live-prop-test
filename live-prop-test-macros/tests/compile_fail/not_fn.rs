use live_prop_test_macros::live_prop_test;

#[live_prop_test]
struct NotAFunction;

fn main() {}
