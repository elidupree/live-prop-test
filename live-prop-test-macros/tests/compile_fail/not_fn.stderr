error: #[live_prop_test] can only be applied to a fn item, impl item, or trait item
       
         = help: if it looks like it's already applied to one of those things, maybe there's a syntax error in the body? live_prop_test can only recognize what it is if it's syntactically valid.
         = help: try commenting out the #[live_prop_test] attribute to make it easier to find the syntax error.
       
 --> $DIR/not_fn.rs:3:1
  |
3 | #[live_prop_test]
  | ^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `live_prop_test` (in Nightly builds, run with -Z macro-backtrace for more info)
